registry/registry_db.c

```diff
@@ -89,7 +95,7 @@ static inline int registry_person_save(void *entry, void *file) {
     );
 
     if(ret >= 0) {
-        //int ret2 = dictionary_get_all(p->person_urls, registry_person_save_url, fp);
+        //int ret2 = dictionary_walkthrough(p->person_urls, registry_person_save_url, fp);
```

Comment:
@ktsaou I observed that you renamed a commented code for we keep compatibility, but I am thinking the previous code was already commented, because we were using `avl_transverse`, should we not remove this comment?

Reply:
I saw it too. But I can't remember why I commented that part back then. I commented it for a reason. But I haven't written why. So, when we touch that part again, we should know that the commented code was not good for some reason.

Let's leave this comment there, it may be useful if ever we want to revisit that part of the code.

Reply:
All right, I will mark this as resolved, and I will take a look in the last commits.